internal/services/qumulo/qumulo_file_system_resource_test.go

```diff
@@ -0,0 +1,219 @@
+package qumulo_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/qumulostorage/2024-06-19/filesystems"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+)
+
+type FileSystemTestResource struct{}
+
+func TestAccQumuloFileSystem_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_qumulo_file_system", "test")
+	r := FileSystemTestResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep("admin_password", "email"),
```

Comment:
I'd expect the API to not return the password, but why doesn't it return `email`

Reply:
The service team confirms that email addresses are treated as Personally Identifiable Information (PII) and are never stored on Azure. So the current behavior of not returning email addresses in the response is by design.